tests/unit/type/list.tcl

```diff
@@ -15,16 +15,171 @@ proc write_big_bulk {size} {
     r read
 }
 
-# SORT which attempts to store an element larger than 4GB into a list.
-# Currently unsupported and results in an assertion instead of truncation
 start_server [list overrides [list save ""] ] {
 
+    r config set list-compress-depth 2
```

Comment:
don't you also need to set `list-max-ziplist-size` so that each entry is in it's own node?
maybe add some comments explaining what you aim to test in each of these tests and how.